Headers/AppKit/GSSVGImageRep.h

```diff
@@ -0,0 +1,47 @@
+/* Definition of class GSSVGImageRep
+   Copyright (C) 2019 Free Software Foundation, Inc.
+   
+   By: heron
```

Comment:
I need to remove it.  Since there is no additional information to be retrieved for SVG files I didn't implement this.   Also I did the filtering by putting the specific implementations first in the list of image reps to be considered.   I will fill my name in.  Sorry, it's a side effect of the script I used to generate the skeletons.

Reply:
I made no change to NSBitmapImageRep+BMP... so I don't understand.

Reply:
That file is just a copy of NSBitmapImageRep+PNG.m, please remove it.

Reply:
Removed.